doc/md/CLI/getting_started.mdx

```diff
@@ -84,5 +84,55 @@ curl -LO https://release.ariga.io/atlas/atlas-windows-amd64-v0.0.1.exe
 Move the atlas binary to a file location on your system PATH.
 
 
+</TabItem>
+</Tabs>
+
+## Schema Inspection
+
+<Tabs
+defaultValue="mysql"
+values={[
+{label: 'MySQL', value: 'mysql'},
+{label: 'PostgreSQL', value: 'postgres'},
+]}>
+<TabItem value="mysql">
+
+Inspect and save output to a schema file.
+```shell
+atlas schema inspect -d "mysql://root:pass@tcp(localhost:3306)/example" >> atlas.hcl
+```
+
+</TabItem>
+<TabItem value="postgres">
+
+Inspect and save output to a schema file.
+```shell
+atlas schema inspect -d "postgres://root:pass@0.0.0.0:5432/example?sslmode=disable" >> atlas.hcl
+```
+
+</TabItem>
+</Tabs>
+
+## Apply change to Schema
+
+<Tabs
+defaultValue="mysql"
+values={[
+{label: 'MySQL', value: 'mysql'},
+{label: 'PostgreSQL', value: 'postgres'},
+]}>
+<TabItem value="mysql">
+
+```shell
+atlas schema apply -d "mysql://root:pass@tcp(localhost:3306)/example" -f atlas.hcl
+```
+
+</TabItem>
+<TabItem value="postgres">
+
+```shell
+atlas schema apply -d "postgres://root:pass@0.0.0.0:5432/example?sslmode=disable" -f atlas.hcl
```

Comment:
add a warning/caveat that sslmode disable is not recommended in prod

Reply:
👍

Reply:
Your call but I think for the user that wants to experiment with something new it's best to reduce all friction . Setting up a local dB to play with is friction something I can copy paste to safely plan with can help.
Ultimately, whatever you think